lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp

```diff
@@ -991,6 +991,10 @@ status_code_e amf_smf_handle_ip_address_response(
         OAILOG_ERROR(LOG_AMF_APP, "Create IPV4V6 Session \n");
       }
     }
+  } else {
```

Comment:
On failing to allocate IP or on reception of gRPC status as "not ok"
1. Then amf/smf should not update the context with received IP address as it is done in line, https://github.com/magma/magma/blob/69d17a7a6640fa1161dc7cc3c38e47d0d975f44f/lte/gateway/c/core/oai/tasks/amf/amf_smf_send.cpp#L932
2. This condition shall be present at start of function.
3. At UE, on reception of pdu_session_establishment_reject for initial request type, then UE shall not create pdu session. So I think amf/smf should clear the session contexts. Correct me if I am wrong

Reply:
Addressed the comments.